internal/terraform/rootmodule/root_module_manager.go

```diff
@@ -78,15 +78,15 @@ func (rmm *rootModuleManager) AddRootModule(dir string) error {
 
 	_, exists := rmm.rms[dir]
 	if exists {
-		return fmt.Errorf("rootModule %s was already added", dir)
+		return fmt.Errorf("root module %s was already added", dir)
```

Comment:
Not sure if it matters, but should this just be a noop instead of an error?

Reply:
I suppose it may come down to it eventually, but I think we do not ever add the same module twice, and if we are then I'd like to understand why first, before allowing it.